### PR TITLE
Create fix_320_start_dates.sql

### PR DESCRIPTION
**Pull request by scottty881**

### CODE REVIEW

File: dmls_to_run/fix_320_start_dates.sql [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+AND value::int IN 
+  (
+    SELECT sub_groups.id 
+    FROM sub_groups INNER JOIN
+    groups ON groups.id = sub_groups.group_id WHERE groups.name = '320'
+  )
+  RETURNING *;
+  
+  
+UPDATE
```
Comment (scottty881) [inline dmls_to_run/fix_320_start_dates.sql]: 114 rows

File: dmls_to_run/fix_320_start_dates.sql [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+subscriber_data_values 
+SET effective_date = '2017-10-01'
+WHERE field = 'sub_group_id' 
+AND value::int IN
```
Comment (scottty881) [inline dmls_to_run/fix_320_start_dates.sql]: 326, prob cuz of dups

Comment (yjhuoh) [reply dmls_to_run/fix_320_start_dates.sql]: 326 isn't triple of 114. need a better explanation for this?

Comment (scottty881) [reply dmls_to_run/fix_320_start_dates.sql]: i think theyre separate issues, lets not combine them

Comment (scottty881) [reply dmls_to_run/fix_320_start_dates.sql]: we know for a fact every effective date for this group should be 10-1

**Pull request closed without merging**